LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/CaseInsensitiveEnvironmentVariables.cs

```diff
@@ -49,7 +49,7 @@ public uint GetEnvVar(string key, uint defaultValue)
             return value;
         }
 
-        public double GetEnvVar(string key, double defaultValue)
+        public double? GetEnvVar(string key, double? defaultValue)
```

Comment:
Is this method being used in a scenario that a defaultValue other than null is set?
If it returns nullable of double you could return null in case you cannot get a valid value.

Reply:
It is currently not used for another default value, however some other methods signature are. I thought keeping a similar signature between method might be good, additionally maybe the functionality could be needed in the future.

Reply:
Perhaps add a default value:
`public double? GetEnvVar(string key, double? defaultValue = null)`

this way callers don't have to have to set a null nullable value as the default.

Reply:
Good idea will do